lib/util/updateProperty.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+export function updateProperty<T, K extends keyof T>(
+  obj: T, k: K, update: (v: T[K], k: K) => T[K])
+  : void {
```

Comment:
wouldn't be preffered to return the value instead of doing mutations on the existing object ?

Reply:
also cant we just use [https://lodash.com/docs/4.17.10#update](https://lodash.com/docs/4.17.10#update) ?

---
In reply to: [197904198](https://github.com/Azure/oav/pull/265#discussion_r197904198) [](ancestors = 197904198)

Reply:
`lodash.update` doesn't provide type safety.

Reply:
maybe a comment on the function to explain the type restrictions

---
In reply to: [197939074](https://github.com/Azure/oav/pull/265#discussion_r197939074) [](ancestors = 197939074)